treeherder/webapp/api/push.py

```diff
@@ -75,17 +74,11 @@ def list(self, request, project):
         if search_param:
             repository = Repository.objects.get(name=project)
             filtered_commits = (
-                Commit.objects.annotate(
-                    search=SearchVector(
-                        "revision", "author", Substr("comments", 1, 100000), config="english"
-                    )
-                )
-                .filter(
-                    search=SearchQuery(search_param, config="english"),
+                Commit.objects.filter(
```

Comment:
we can probably switch to the code that you tried before and that avoids the subquery.